File: trunk/JAVA/PokerProtocol/src/bluffinmuffin/protocol/commands2/lobby/AbstractLobbyResponse.java
package bluffinmuffin.protocol.commands2.lobby;

import bluffinmuffin.protocol.commands2.AbstractCommandResponse;

public abstract class AbstractLobbyResponse<T extends AbstractLobbyCommand> extends AbstractCommandResponse<T>
{
    
    public AbstractLobbyResponse(T command)
    {
        super(command);
    }
    
    @Override
    protected <T2> void append(StringBuilder sb, T2 thing)
    {
        sb.append(thing);
        sb.append(AbstractLobbyCommand.Delimitter);
    }
    
}

File: C#/BluffinMuffin.Protocol.Commands/Lobby/AbstractLobbyResponse.cs
using Com.Ericmas001.Net.Protocol.JSON;

namespace BluffinMuffin.Protocol.Commands.Lobby
{
    public abstract class AbstractLobbyResponse<T>: AbstractJsonCommandResponse<T>
        where T : AbstractLobbyCommand
    {
        public AbstractLobbyResponse()
        {
        }
        public AbstractLobbyResponse(T command)
            : base(command)
        {
        }
    }
}
